functions.js

```diff
@@ -1,3 +1,25 @@
+// remove once code reviewed
+const testObjectForCodeReview = {
+    name: 'England',
+    code: 'ENG',
+    anagram: 'nldegan'
+}
+/**
+ * function to get display country anagram and country code to html
```

Comment:
Didn't make sense to us, maybe: 'function to display anagram & country code to front end'